db/models/models.js

```diff
@@ -88,3 +88,31 @@ exports.insertCommentByReviewId = (review_id, newComment) => {
       return res.rows[0];
     });
 };
+
+exports.updateVotes = (review_id, inc_votes) => {
+  if (!inc_votes) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad request"
+    });
+  }
+  return checkExists("reviews", "review_id", review_id)
```

Comment:
Do you need to check if the review exists here?

Reply:
Not sure what you mean by this bit